applications/nrf_desktop/src/util/chmap_filter/include/chmap_filter.h

```diff
@@ -0,0 +1,255 @@
+/*
```

Comment:
Is this library specific to nrf_desktop or could belong in nrf/subsys/bluetooth ?

Reply:
It is not functionally restricted to nrf_desktop, but all the default parameter values are tuned for the HID device use case (high packet rate). Any BLE Central device using the nrfx controller could in theory use this as a means to avoid busy RF channels. The plan was to introduce it under nrf_desktop at first, and rather move it later if desired